google/cloud/storage/client.h

```diff
@@ -2422,12 +2396,12 @@ class Client {
    *     Cloud Pub/Sub service.
    */
   template <typename... Options>
-  void DeleteNotification(std::string const& bucket_name,
-                          std::string const& notification_id,
-                          Options&&... options) {
+  Status DeleteNotification(std::string const& bucket_name,
```

Comment:
Per our discussion offline, might we want to return `StatusOr<void>` here?

Reply:
Done.